### PR TITLE
Lesson 1-1 Project Setup and React basics

### PR DESCRIPTION
**Pull request by gkojunior**
Now the user can view a list of todo list items

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    id: 1,
+    title: "JavaScript Basic"
+  },
+  {
+    id: 2,
+    title: "Become a React.JS Developer"
+  },
+  {
+    id: 3,
+    title: "Take Node.JS Coarse"
```
Comment (dorianmariecom) [inline src/App.js]:
```suggestion
    title: "Take Node.JS Course"
```

;)

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const todoList = [
```
Comment (dorianmariecom) [inline src/App.js:3]: should be `items`

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
```
Comment (dorianmariecom) [inline src/index.js:10]: Should you be using `ReactDOM.createRoot`? https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis

**Review by dorianmariecom — COMMENTED** [review]
Good start, I would not rename README.md to README.old.md but edit it instead.

Also you could format your code using [prettier](https://prettier.io/) which is the standard for JavaScript.

You pull request description should be focused on the user perspective, e.g.

> Now the user can view a list of todo list items

Also this is mostly generated files, you could edit them to be specific to your application (html title, package name, comments in index.html, html description, favicon)

Personnally I much prefer using esbuild than webpack but I guess this tutorial uses webpack so better stick with it.

File: src/App.js (hunk 2 above)
Comment (gkojunior) [reply src/App.js:3]: Hey Dorian, thanks you for the advice. I was able to make the changes. The lesson instructed us to store it in a variable called todoList.

**Review by gkojunior — COMMENTED** [review]
Dorian, thank you for your advice, changes have been made.

File: src/App.js (hunk 2 above)
Comment (gkojunior) [reply src/App.js:3]: I also have prettier installed, I didn't know it wasn't applying to the code. I went back to the extenttion and I ran this code in my terminal "npm install prettier -D --save-exact" is there something in the setting in which I should change

Comment (dorianmariecom) [reply src/App.js:3]: you can do `npm install -g prettier` then `prettier --write .`

**Pull request merged**